AestraUI/Widgets/AestraEQEditor.cpp

```
@@ -14,6 +15,108 @@ namespace AestraUI {
+        NUIColor(0.48f, 0.82f, 1.0f, 1.0f),
+        NUIColor(0.62f, 0.74f, 1.0f, 1.0f),
+        NUIColor(0.74f, 0.67f, 1.0f, 1.0f),
+        NUIColor(0.96f, 0.72f, 0.52f, 1.0f),
+        NUIColor(0.90f, 0.60f, 0.40f, 1.0f),
+        NUIColor(0.88f, 0.56f, 0.76f, 1.0f),
+        NUIColor(0.78f, 0.76f, 1.0f, 1.0f),
+    };
+    return colors[index % (sizeof(colors) / sizeof(colors[0]))];
+}
+
+const char* shortTypeLabel(uint32_t type) {
```

Cppcheck (2.20.0)

[style] The function 'shortTypeLabel' is never used.

(unusedFunction)